index.js

```diff
@@ -28,6 +28,7 @@ const {
 } = require("./utils/table.js");
 const xcolor = cli.flags.xcolor;
 const sortBy = cli.flags.sort;
+const limit = cli.flags.limit;
```

Comment:
If the limit is negative, some unexpected results might be generated: `slice(0, -10)` for example might be a bit odd. Should we limit this to positive integers? Is it possible to do this in the `cli` utility?

Reply:
How do you propose we solve this @ahmadawais? Should we default to a given value if the number is negative, or should we throw an error? Can this be handled in the CLI utility?